### DISCUSSION TITLE
Unable to view private GitHub Pages

### DISCUSSION THREAD

**Discussion by forky2** [community/community#23146] [Enterprise]
<p>Hello,</p>
<p>I’ve got a GitHub Enterprise Cloud account and I’m struggling to get private pages to work. In a test project in an Organisation, I’ve set the repo to private and I’ve created a docs directory with an index.html file, and then I’ve enabled Pages for the /docs directory in the master branch. Visibility of Pages is set to private as per my requirements, and ostensibly it has been built correctly. I’ve got a <a href="https://adjective-noun-randomid.pages.github.io/" rel="noopener nofollow ugc">https://adjective-noun-randomid.pages.github.io/</a> address, but when I visit it I just get redirected to the GitHub settings page for my email details.</p>
<p>Interestingly if I change visibility (of Pages, not the project) to public, then I get an address of the form <a href="https://organisation.github.io/projectname" rel="noopener nofollow ugc">https://organisation.github.io/projectname</a>, and this displays the html fine. Changing back to private, it’s broken again.</p>
<p>So something appears to be broken with the ‘private’ visibility functionality.</p>
<p>Any ideas please?</p>

**Comment by forky2** [comment, marked as answer]
<p>I’ve got a solution to this thanks to Allie at GitHub support.</p>
<p>My problem, it turned out, was due to by browser cache needing to be cleared. For some reason it wasn’t properly authenticating me to access the page. First clue was that it was working from a private browser session.</p>
<p>Hope this helps anyone that sees the same weird behaviour.</p>
<p>Cheers.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I was just looking into this when I saw the ticket was resolved. Thank you for sharing the solution - I can guarantee others will have the same question :slight_smile:</p>

**Comment by galaxyeden** [comment]
<p>Hi <a class="mention" href="/forky2">@forky2</a> - thanks for letting us know on here as well for others, and please don’t hesitate to let me know if you have any other questions!</p>
